src/routes/useNavigate.tsx

```diff
@@ -0,0 +1,15 @@
+import { useNavigate as useReactNavigate, NavigateOptions } from "react-router-dom";
+
+import { RoutePath } from "./Routes";
+
+const useNavigate = () => {
+  const navigate = useReactNavigate();
+
+  const typedNavigate = (to: RoutePath, options?: NavigateOptions) => {
+    navigate(to, options);
+  };
```

Comment:
```suggestion
  const ReactNavigate = useReactNavigate();

  const navigate = (to: RoutePath, options?: NavigateOptions) => {
    ReactNavigate(to, options);
  };
```

typedNavigate 네이밍이 좀 애매한것 같은데 위처럼 바꾸는건 어떨까요?

Reply:
넵 좋습니다~